docs/externalGrading.md

```diff
@@ -70,6 +70,8 @@ External grading configuration is done on a per-question basis. All configuratio
 
 * `enableNetworking`: Allows the container to access the public internet. This is disabled by default to make secure, isolated execution the default behavior.
 
+* `environment`: Environment variables to set inside the grading container. Set variables using `{"VAR": "value", ...}`, and unset variables using `{"VAR": null}` (no quotes around `null`).
```

Comment:
Scripts may respond differently to the cases of a variable being not declared at all vs. assigned a blank value. For example, Node differentiates these as `undefined` and `''`. It may also be desired to hide suggestive variable names leftover from deployment or something.

At any rate, since the environment (e.g. bash `unset`) and Docker allow it, it makes sense to me to support it.